public/stockfish-worker.ts

```
@@ -0,0 +1,36 @@
+// public/stockfish-worker.js (Move this file to public directory)
+
+// Use importScripts instead of import
+importScripts('./stockfish.js');
+import { Stockfish } from 'stockfish.js';
+// or
+import * as StockfishModule from 'stockfish.js';
+const Stockfish = StockfishModule.default || StockfishModule;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Stockfish'. Consider to delete it or rename it.

'Stockfish' is defined here:


(lint/suspicious/noRedeclare)